cli/src/lib/npm/__tests__/npmLibDefs-test.js

```diff
@@ -197,11 +199,18 @@ describe('npmLibDefs', () => {
         const pkgName = 'flow-bin';
         const pkgVersion = 'github:flowtype/flow-bin';
         const flowVersion = {kind: 'all'};
+        const skipCache = false;
 
         let filtered;
         let error;
         try {
-          filtered = await findNpmLibDef(pkgName, pkgVersion, flowVersion);
+          filtered = await findNpmLibDef(
+            pkgName,
+            pkgVersion,
+            flowVersion,
+            undefined,
+            skipCache,
+          );
```

Comment:
Since `skipCache` now has a default value, I don't think you need to explicitly pass it here (same goes for the previous test)

Reply:
Yeah makes sense. Let me fix that up